test_args.sh

```diff
@@ -141,4 +141,13 @@ res=0
 ./livepeer -broadcaster -authWebhookUrl http\\://host/ || res=$?
 [ $res -ne 0 ]
 
+# exit early if maxSessions less or equal to zero
+res=0
+./livepeer -maxSessions -1 || res=$?
```

Comment:
I know that the `-maxSessions` check exits early, but this would fail anyway since the node type is not specified. Maybe add a `-broadcaster` to this and the following test?